x/project/handler.go

```diff
@@ -194,8 +222,21 @@ func handleMsgCreateAgent(ctx sdk.Context, k Keeper, bk bank.Keeper, msg MsgCrea
 	if err != nil {
 		err.Result()
 	}
-
-	return sdk.Result{}
+	ctx.EventManager().EmitEvents(sdk.Events{
+		sdk.NewEvent(
+			types.EventTypeUpdateProjectStatus,
+			sdk.NewAttribute(types.AttributeKeyTxHash, msg.TxHash),
+			sdk.NewAttribute(types.AttributeKeySenderDid, msg.SenderDid),
+			sdk.NewAttribute(types.AttributeKeyProjectDid, msg.ProjectDid),
+			sdk.NewAttribute(types.AttributeKeyData, msg.Data.AgentDid),
```

Comment:
Please add `msg.Data.Role` as an attribute. And then the attribute keys would be: `AttributeKeyAgentDid` and `AttributeKeyAgentRole`.

Reply:
ok. missed to update in the first MR review on AttributeKeyAgentRole.
will update the code and push